src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -172,28 +202,76 @@ const MenuContent = ({ programName }: { programName: string }) => {
 
 	return (
 		<>
-			{/** Register Samples */}
-			<Link href={`/submission/program/${programName}/registration`}>
-				<MenuItem
-					level={3}
-					content={
-						<StatusMenuItem>
-							Register Samples
-							{statusIcon}
-						</StatusMenuItem>
-					}
-					selected={pathnameLastSegment === 'registration'}
-				/>
+			{/** Dashboard */}
+			<Link
+				as={PROGRAM_DASHBOARD_PATH.replace(PROGRAM_SHORT_NAME_PATH, programName)}
+				href={PROGRAM_DASHBOARD_PATH}
+			>
+				<MenuItem level={3} content="Dashboard" selected={PROGRAM_DASHBOARD_PATH === pathname} />
```

Comment:
Let's ensure this works as expected. 

```typescript
export const SUBMISSION_PATH = `/submission`;
export const PROGRAM_SHORT_NAME_PATH = `[shortName]`;
export const PROGRAM_DASHBOARD_PATH = `${SUBMISSION_PATH}/program/${PROGRAM_SHORT_NAME_PATH}/dashboard`;
```

so `PROGRAM_DASHBOARD_PATH` will end up as `/submission/program/[shortName]/dashboard`

Nextjs is routing using `[..foo..]` as dynamic values 
this path constant (and any others that interpolate a dynamic path string) will never match an actual `pathName` unless you interpolate the dynamic segment with the path constant you're checking against

Reply:
these constants come in handy for a usage like: 
```javascript
router.push(PROGRAM_DASHBOARD_PATH.replace(PROGRAM_SHORT_NAME_PATH, shortName));
```
but you're always gonna have to get the dynamic route part injected one way or another

Reply:
see above discussion, updated

Reply:
`<MenuItem level={3} content="Dashboard" selected={PROGRAM_DASHBOARD_PATH === pathname} />`
 
this `selected` prop for example will be `selected={'/submission/program/[shortName]/dashboard' === '/submission/program/CIA-IE/dashboard'`

`PROGRAM_DASHBOARD_PATH` will always have the `[shortName]` dynamic placeholder in the string until you substitute it dynamically with the actual pathname of a page.

this is introducing a bug.

Reply:
Updated in: https://github.com/icgc-argo/rdpc-ui/pull/114/commits/afc07b6f8dcaba6b6b5abb6fa37ee6c2dc80fde3